src/components/input-field/input-field.stories.js

```diff
@@ -1,23 +1,39 @@
 import { html } from "lit-html";
 import "../input-field";
+import "../button";
+import "../link";
+import "../form-message";
 import styles from "./styles.scss";
+import buttonStyles from "../button/styles.scss";
+import linkStyles from "../link/styles.scss";
+import formMessageStyles from "../form-message/styles.scss";
 import {
   stylesheet,
   docsIntro,
   CATEGORIES,
 } from "../../../.storybook/utils.js";
+// import { args, argTypes } from "../form-validation/config";
+// import { VALIDATION_TYPE } from "../form-validation/enum";
 
 const defaultArgs = {
```

Comment:
Apart `defaultArgs` object aanmaken is enkel een beetje verwarrend voor mij, waarom niet gewoon die props onder args? Maw, waarom zijn die 'default' en de rest niet?

Reply:
Fixed.